installers/rpm-code/codes/rsbeams.sh

```diff
@@ -12,7 +12,7 @@ declare -a _rsbeam_codes=(
 rsbeams_main() {
     rsbeams_init_vars
     codes_dependencies common ml
-    local r
+    declare r
     for r in "${_rsbeam_codes[@]}"; do
```

Comment:
If you want to protect this, you have to: `${_rsbeam_codes[@]+"${_rsbeam_codes[@]}"}`